tests/handlers/records-query.spec.ts

```diff
@@ -735,6 +1185,20 @@ export function testRecordsQueryHandler(): void {
         expect(replyToQuery.status.code).to.equal(200);
         expect(replyToQuery.entries?.length).to.equal(1);
         expect((replyToQuery.entries![0].descriptor as RecordsWriteDescriptor).schema).to.equal('https://schema2');
+
+        // explicitly for published records
+        const anonymousQueryUnpublished = await TestDataGenerator.generateRecordsQuery({
+          anonymous : true,
+          filter    : { dateCreated: { from: '2000-01-01T10:20:30.123456Z' }, published: true }
+        });
+        // sanity check
+        expect(anonymousQueryUnpublished.message.authorization).to.not.exist;
+
+        // should not return any records
```

Comment:
Sanity: says should not return any records, but checks for entry count = 1.

Reply:
Good catch! I renamed the variables too, I had this testing the 401 but then moved to a more explicit `should return 401 for anonymous queries that filter explicitly for unpublished records`, forgot to rename the comment/variables for the positive check.